styles/pa11y.scss

```diff
@@ -1,13 +0,0 @@
-a {
-    color: #2770af;
-}
-#maincontent ul > button {
-    color: #001731;
-}
-footer li > a {
-    color: #0061c9;
-}
-
-#main-content time {
-    color: #6c7789;
-}
```

Comment:
Was this file just leftover from trying to do it the "proper" way?  Or are these actually no longer needed?

Reply:
Yeah they were leftovers. I filed this issue with Pa11y because I'm curious if they can give any insight.

https://github.com/pa11y/pa11y/issues/654